src/controller/CHIPDeviceController.h

```diff
@@ -297,6 +297,10 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public Rendezvous
        If no device is currently being paired, this value will be kNumMaxPairedDevices.  */
     uint16_t mDeviceBeingPaired;
 
+    /* This is a temporary flag, when using IP rendezvous, we need to disable network provisioning.
+       In the future, network provisioning will no longer be a part of rendezvous procedure. */
+    bool mIsIPRendezvous;
```

Comment:
Why do we need that exactly ?
Can't we just start the device on SoftAP mode, and then do the same workflow that is used for BLE ? Maybe I'm missing something here.

Reply:
Of course, BLE and IP rendezvous should share the same procedure, this flag will be removed after we reach that timepoint. Currently, when doing rendezvous over BLE, we will also do network provisoning, this flag is used for not doing network prov over IP rendezvous since they are clusters. I will add TODO here to explain this more clearly.